static/js/components/CompactPostDisplay.js

```diff
@@ -134,11 +135,11 @@ export class CompactPostDisplay extends React.Component<Props> {
               </div>
             </div>
           </div>
-          {post.url || post.thumbnail ? (
+          {post.post_type === LINK_TYPE_LINK || post.thumbnail ? (
```

Comment:
Take or leave since it isn't directly related to this PR: it really seems like this 'or' clause is unnecessary. The only thing necessarily in common with link-type posts and posts that have a `thumbnail` value is the wrapper div with the `column2` class, and even then they differ in that a thumbnail-ed post will have an additional class set. Seems like it should just be 2 separate if's.